docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -252,10 +252,11 @@ see its help page:
 .. _Wheels: python-wheels
 .. _Linux packages: linux-packages
 .. _Create the queue repository: https://help.github.com/articles/creating-a-new-repository
-.. _TravisCI: https://travis-ci.org/getting_started
+.. _TravisCI: https://travis-ci.com/getting_started
```

Comment:
Looks like this is a 404 currently?

Reply:
Good catch! It looks like this is supposed to be `getting-started` instead of `getting_started`. This may have been a typo, or maybe the URL changed recently.

Reply:
This still needs fixing apparently?

Reply:
Sorry, my mistake.